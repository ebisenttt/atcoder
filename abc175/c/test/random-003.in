-209096170148419 551256675322404 97407244772232
