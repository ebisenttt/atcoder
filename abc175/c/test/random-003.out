14281680603955
